### PR TITLE
chore: add new attribute to differentiate first purchase in sap

### PR DESCRIPTION
**Pull request by msimionati**
add new attribute to differentiate first purchase in sap

### CODE REVIEW

File: Doppler.Sap/Mappers/SaleOrderMapper.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ public static SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRe
             {
                 NumAtCard = billingRequest.PurchaseOrder ?? "",
                 U_DPL_RECURRING_SERV = billingRequest.IsPlanUpgrade ? "N" : "Y",
+                U_DPL_FIRST_PURCHASE = billingRequest.IsFirstPurchase ? "Y" : "N",
```
Comment (jhoffman-ms) [inline Doppler.Sap/Mappers/SaleOrderMapper.cs:19]: This class (```SaleOrderMapper```) can be removed. Could you please check if the class is used?. For me was replaced by ```BillingForArMapper```.

Comment (msimionati) [reply Doppler.Sap/Mappers/SaleOrderMapper.cs:19]: Ok, I see it

**Review by bamaro-dev — COMMENTED** [review]
✔️

**Comment by nreal** [comment]
👍

**Pull request merged**